sentry-android-replay/src/main/java/io/sentry/android/replay/DefaultReplayBreadcrumbConverter.kt

```diff
@@ -0,0 +1,149 @@
+                breadcrumb.category == "device.orientation" -> {
+                breadcrumbCategory = breadcrumb.category!!
+                val position = breadcrumb.data["position"]
+                if (position == "landscape" || position == "portrait") {
+                    breadcrumbData["position"] = position
+                } else {
+                    return null
+                }
+            }
+
+            breadcrumb.type == "navigation" -> {
+                breadcrumbCategory = "navigation"
+                breadcrumbData["to"] = when {
+                    breadcrumb.data["state"] == "resumed" -> (breadcrumb.data["screen"] as? String)?.substringAfterLast('.')
+                    "to" in breadcrumb.data -> breadcrumb.data["to"] as? String
+                    else -> null
+                } ?: return null
+            }
+
+            breadcrumb.category == "ui.click" -> {
+                breadcrumbCategory = "ui.tap"
+                breadcrumbMessage = (
+                    breadcrumb.data["view.id"]
+                        ?: breadcrumb.data["view.tag"]
+                        ?: breadcrumb.data["view.class"]
+                    ) as? String ?: return null
+                breadcrumbData.putAll(breadcrumb.data)
+            }
+
+            breadcrumb.type == "system" && breadcrumb.category == "network.event" -> {
+                breadcrumbCategory = "device.connectivity"
+                breadcrumbData["state"] = when {
+                    breadcrumb.data["action"] == "NETWORK_LOST" -> "offline"
+                    "network_type" in breadcrumb.data -> if (!(breadcrumb.data["network_type"] as? String).isNullOrEmpty()) {
+                        breadcrumb.data["network_type"]
+                    } else {
+                        return null
+                    }
+
+                    else -> return null
+                }
+            }
+
+            breadcrumb.data["action"] == "BATTERY_CHANGED" -> {
+                breadcrumbCategory = "device.battery"
+                breadcrumbData.putAll(
+                    breadcrumb.data.filterKeys { it == "level" || it == "charging" }
+                )
+            }
+
+            else -> {
+                breadcrumbCategory = breadcrumb.category
+                breadcrumbMessage = breadcrumb.message
+                breadcrumbLevel = breadcrumb.level
+                breadcrumbData.putAll(breadcrumb.data)
+            }
+        }
+        if (!breadcrumbCategory.isNullOrEmpty()) {
+            rrwebBreadcrumb = RRWebBreadcrumbEvent().apply {
+                timestamp = breadcrumb.timestamp.time
+                breadcrumbTimestamp = breadcrumb.timestamp.time / 1000.0
+                breadcrumbType = "default"
+                category = breadcrumbCategory
+                message = breadcrumbMessage
+                level = breadcrumbLevel
+                data = breadcrumbData
+            }
+        }
+        return rrwebBreadcrumb
```

Comment:
looks like rrwebBreadcrumb is always `null` anyway. How about this:
```suggestion
        return if (!breadcrumbCategory.isNullOrEmpty()) {
            RRWebBreadcrumbEvent().apply {
                timestamp = breadcrumb.timestamp.time
                breadcrumbTimestamp = breadcrumb.timestamp.time / 1000.0
                breadcrumbType = "default"
                category = breadcrumbCategory
                message = breadcrumbMessage
                level = breadcrumbLevel
                data = breadcrumbData
            }
        } else {
            null
        }
```